### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by Linjieqiang** [SimosMCmuffin/FlexiBMS_hardware#2]
Hello Simos.

Happy to see your BMS project.

1. Why not use the TI BQ76940 not use LTC6803 as the Analog front chip to catch the each cell voltage and current and temperature?
2. Why not use the TI BQ76200 to controll the power switch?

**Comment by SimosMCmuffin** [comment]
Good questions.

1. **Why not use TI's BQ769XX series ICs for the cell stack monitoring?** I had in fact in the beginning of the development looked at the before mentioned TI's chips, as a lower priced alternative to the LTC6803 series, but discovered the disappointing fact, that the overall cell support was less flexible with the TI's chips. I have pasted below a screenshot from the BQ769XX datasheet which shows the problem, there is no IC model that would support 4-12 amount of series cells and that is why I went with the LTC6803, because it gave the most flexibility for the series cells configuration.

![image](https://user-images.githubusercontent.com/9884848/40898954-e93f5770-67cc-11e8-96d4-18c1fb3c37b6.png)

2. **Why not use TI's BQ76200 for controlling the charge FET?** Remember that there is no discharge control on the main module, it would only be controlling a single high-side N-channel FET ideally, but that doesn't really make that much sense in the end. The charging currents we're talking about are low enough that a decently sized P-channel FET can be used with a simple low side N-channel FET to activate it. So IMO it's cheaper, less complex and uses less board space to go with the P-channel implementation.
